distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -958,9 +958,8 @@ type DB_Table
              t2.read
     limit : Integer -> DB_Table
     limit self max_rows:Integer=1000 =
-        Feature.Sample.if_supported_else_throw self.connection.dialect "limit" <|
-            new_ctx = self.context.set_limit max_rows
-            self.updated_context new_ctx
+        new_ctx = self.context.set_limit max_rows
```

Comment:
Why is this check removed?

Reply:
I was getting failures like:
```
        Reason: An unexpected dataflow error (limit not yet supported for DB backend. Use .read to download the data and use this feature.) has been matched (at X:\NBO\enso\test\Table_Tests\src\Database\Common\Audit_Spec.enso:71:13-49).
        at <enso> Feature.if_supported_else_throw<arg-2>(X:\NBO\enso\built-distribution\enso-engine-0.0.0-dev-windows-amd64\enso-0.0.0-dev\lib\Standard\Database\0.0.0-dev\src\Feature.enso:60:13-73)
```
when invoking `DB_Table.read`.

I'm surprised it was working at all in the first place, because our `read` by default relies on `limit`. It is actually an interesting thing to investigate.

But overall - as currently implemented - `limit` is an intrinsic part of the `read` operation, so it must be supported if we want proper `read`. Thus I don't think it should be hidden behind `Sample` flag. We could hide it if we change how `read` works, but I'm not convinced we should.